### ISSUE TITLE
[$15][MSFT-67] [Member] Missing form Label in Search Field

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2781]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Inspect Element
3. Observe the search field label.

**Expected Result**
Form Label should be present in Search Field

**Actual Result**
Form Label is missing in Search Field. 
<input type="search" placeholder="Search" class="text-ctrl" name="s" id="search-query-mobile" autocomplete="off">
<div class="search-clear">

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2067

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096287 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096287 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu I am not able to understand this, can you please give more detail on this one?

**Comment by veshu** [comment]
@mishacucicea Please check here

**Comment by mishacucicea** [comment]
@asadath1395 Take a look here: https://www.w3.org/WAI/tutorials/forms/labels/

**Comment by asadath1395** [comment]
@mishacucicea Even this is part of homepage i guess

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097834 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
